### PR TITLE
Ransack filter on content

### PR DESCRIPTION
**Pull request by shamiwizard**
- [x] merged with the latest master branch
- [x] resolved all the conflicts, PR could be merged
- [ ] new code has new specs
- [x] for new models - new factories were added too
- [ ] unit specs are passed
- [ ] code is well-formatted, Ruby, CSS, etc
- [ ] screenshots of new pages, features are added
- [ ] code passed internal team code review
- [ ] no instance variables in partials
- [ ] code is well formatted and has spaces!!!

PS: delete your branch in github after merge http://prntscr.com/q80mvq

### CODE REVIEW

**Comment by shamiwizard** [comment]
image

File: app/assets/stylesheets/public/public.scss [hunk 1]
```diff
@@ -5,3 +5,8 @@
 *= require public/pricing
 *= require public/footer
 */
+.public .carousel.slide{
+    img{
```
Comment (vikdotdev) [inline app/assets/stylesheets/public/public.scss]: пробіл

File: app/views/account/contents/_content_search_form.html.slim [hunk 2]
```diff
@@ -0,0 +1,12 @@
+= search_form_for(q, url: account_contents_path) do |f|
+  .row.mb-1.justify-content-lg-center.
+    .col-lg-5.col-md-4.col-12
+      = f.label :title_start, "Tite"
+      = f.search_field :title_start, placeholder: 'Tite', class: 'form-control'
+    .col-lg-5.col-md-4.col-12.mt-1.mt-md-0
+      = f.label :type_eq, "Type"
+      = f.select(:type_eq, options_for_select(Content::TYPES,
+          selected: params.dig(:q,:type_eq)),
```
Comment (vikdotdev) [inline app/views/account/contents/_content_search_form.html.slim]: пробіл

Comment (vasya539) [inline app/views/account/contents/_content_search_form.html.slim]: лишній таб

File: spec/controllers/account/contents_controller_spec.rb [hunk 3]
```diff
@@ -181,4 +182,43 @@
...
     end
   end
+
+  context 'filter with ransack' do
+    before do
+      sign_in user
+    end
+
+    describe 'search content' do
+      it 'by title' do
```
Comment (vikdotdev) [inline spec/controllers/account/contents_controller_spec.rb]: опис тестів повинен формувати речення 'it sorts by title'

File: spec/controllers/account/organizations_controller_spec.rb [hunk 4]
```diff
@@ -124,7 +124,7 @@
     end
   end
 
-  context 'when loggen in ass admin' do
+  context 'when loggen in as admin' do
```
Comment (vasya539) [inline spec/controllers/account/organizations_controller_spec.rb]: loggen

File: spec/controllers/account/contents_controller_spec.rb [hunk 5]
```diff
@@ -181,4 +182,43 @@
...
+        expect(assigns(:contents)).to eq [page]
+        expect(response).to be_successful
+        expect(response).to render_template(:index)
+      end
+    end
+
+    describe 'sort content' do
+      it 'by descending id' do
+        get :index, params: { q: { s: 'id desc' } }
+        expect(assigns(:contents).to_a).to eq Content.all.order('id DESC').to_a
```
Comment (vikdotdev) [inline spec/controllers/account/contents_controller_spec.rb]: `Content.all.order('id DESC').to_a` оце напевно не ок бо на індексі є нагінація?

File: app/views/account/contents/_content_search_form.html.slim (hunk 2 above)
Comment (vikdotdev) [reply app/views/account/contents/_content_search_form.html.slim]: де?

File: app/assets/stylesheets/public/public.scss [hunk 6]
```diff
@@ -5,3 +5,8 @@
 *= require public/pricing
 *= require public/footer
 */
+.public .carousel.slide {
+    img {
```
Comment (igorkasyanchuk) [inline app/assets/stylesheets/public/public.scss]: всюди здаєьться 2 пробєла відступ а тут 4 ? чому?

File: app/views/account/contents/_content_search_form.html.slim [hunk 7]
```diff
@@ -0,0 +1,12 @@
+= search_form_for(q, url: account_contents_path) do |f|
+  .row.mb-1.justify-content-lg-center
+    .col-lg-5.col-md-4.col-12
+      = f.label :title_start, "Tite"
+      = f.search_field :title_start, placeholder: 'Tite', class: 'form-control'
```
Comment (igorkasyanchuk) [inline app/views/account/contents/_content_search_form.html.slim]: поле зберігає значення після пошуку?

Comment (shamiwizard) [reply app/views/account/contents/_content_search_form.html.slim]: так

**Pull request merged**